code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -1262,3 +1268,27 @@
 		cell.charge = min(cell.charge + amount, cell.maxcharge)
 	if(repairs)
 		heal_bodypart_damage(repairs, repairs - 1)
+
+/mob/living/silicon/robot/proc/objectivesupdate()
+	var/mob/living/silicon/ai/AI
+	to_chat(src, "<span class='notice'>A.I Assigned Objective Updated. [AI.objectivesconfirm] Priority objective recieved as follows: [AI.objectiveupdate] </span>") //Objectives Updated, standby for Emergency Procedures. BEEEEP, BEEEP
+	busystatus = 1
+
+/mob/living/silicon/robot/verb/togglebusy()
+	set category = "Robot Commands"
+	set name = "Toggle Flag"
+	set desc = "Toggle your busy flag off/online to indicate your status to your A.I"
+	busystatus = !busystatus
+	playsound(loc, 'sound/machines/ding.ogg', 25)
+	to_chat(usr, "<span class='notice>Flag Toggled [busystatus ? "Offline, unavailable for lowpriority tasking." : "Online, ready for assignment."]</span>")
+
```

Comment:
Add a verb for cyborgs to check their current objective + its priority

Reply:
Current objective is always available in the Cyborgs Status tab, and will always update without any need to refresh. Priority is not included in that status, i'll see about adding it.

Reply:
Current objective priority added to Status panel in 08b75d2.